src/Nethermind/Nethermind.AccountAbstraction.Test/UserOperationPoolTests.cs

```diff
@@ -298,6 +301,12 @@ public void Bans_paymaster_if_it_uses_too_much_gas_for_simulation_too_many_times
             accessListSource.AccessList.Returns(new AccessList(
                 new Dictionary<Address, IReadOnlySet<UInt256>> {{new("0x0000000000000000000000000000000000000001"), new HashSet<UInt256> {0}}}));
             
+            IPeerManager peerManager = Substitute.For<IPeerManager>();
+            peerManager.MaxActivePeers.Returns(10);
+            
+            IP2PProtocolHandler p2pProtocolHandler = Substitute.For<IP2PProtocolHandler>();
```

Comment:
i recommend making a seperate test file for p2p op tests, not just expanding on the UserOperationPoolTests.cs